refocus-bdk-server.js

```diff
@@ -111,8 +111,11 @@ if (((logging === 'both') || (logging === 'file')) &&
  */
 function hasAlreadyBeenConsumed(cache, id, updatedAt, botName, eventType) {
   if (!cache) return Promise.resolve(false);
-  return cache.hasBeenConsumed(id, updatedAt, botName,
-    eventType);
+  try {
+    return cache.hasBeenConsumed(id, updatedAt, botName, eventType);
```

Comment:
Looks like test is failing here -  ` UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'hasBeenConsumed' of null`

Reply:
man, completely forgot about this PR here :)